libs/cardano-ledger-core/src/Cardano/Ledger/CertState.hs

```diff
@@ -275,8 +279,76 @@ toPStatePair PState {..} =
   , "deposits" .= psDeposits
   ]
 
+data AnchorDataHash
+
+data Anchor c = Anchor
```

Comment:
Maybe we should put into the BaseTypes module

Reply:
Let's fix the circular dependency. 

Move this instance from `Cardano.Ledger.Orphans`:
```
instance Default Network where
  def = Mainnet
```
into the place where it belongs: `Cardano.Ledger.BaseTypes`

This should make it possible to put `Anchor` into `Cardano.Ledger.BaseTypes` as well